src/main.rs

```diff
@@ -16,7 +17,23 @@ use wrangler::version::background_check_for_updates;
 use anyhow::Result;
 use structopt::StructOpt;
 
-fn main() -> Result<()> {
+fn main() {
+    if let Err(err) = main_handling_errors() {
+        let mut msg = format!("Error: {}", err);
+        for cause in err.chain().skip(1) {
+            write!(msg, "\n\nCaused by:\n    {}", cause).unwrap();
+        }
+        eprintln!("{}", msg);
+
+        let backtrace = err.backtrace().to_string();
+        if backtrace != "disabled backtrace" && backtrace != "unsupported backtrace" {
```

Comment:
Also, this is not a *great* thing to depend on because the Display impl of Backtrace isn't stable. But I didn't see a stable way to get this info.